recipes/gmp/all/test_package/conanfile.py

```diff
@@ -30,6 +34,6 @@ def test(self):
         if can_run(self):
             bin_path = os.path.join(self.cpp.build.bindirs[0], "test_package")
             self.run(bin_path, env="conanrun")
-            if self.options["gmp"].enable_cxx:
+            if self.enable_cxx:
```

Comment:
not very elegant, there is a better alternative based on ctest

Reply:
HI @SpaceIm, Hi @Hopobcn,
Thanks for the suggestion and sample code. I have reworked the testing implementation based on this input.